### ISSUE TITLE
riscvOVPsim targets

### ISSUE THREAD

**Issue by Imperas** [incoresemi/riscv-compliance#5]
Neel - I think you commented when you deleted the device Makefile.includes'- fails because ovpsim requires signatures to be multiple of 16bytes. Can't figure out an override for this.
'
Actually riscvOVPsim does not require it - the compliance suite required it as that was what was in in spike -but a few weeks ago you changed spike and the format in the your compliance suite and probably will break a lot of other peoples flows too.

we tracked the change spike so riscvOVPsim is compatible.

please download latest https://github.com/riscv-ovpsim/imperas-riscv-tests/tree/main/riscv-ovpsim and see the --showoverrides command and you can use the override SignatureGranularity=4 I think.

please try this and then please re-instate the riscvOVPsim device Makefiles so the simulator runs with the changes you are making.

thanks
Simon

**Comment by neelgala** [comment]
@Imperas 

I didn't delete them, but the .gitignore update to stop tracking those folders seems to have made a change that got past me.

Anyway, I have added the target files back. For ovpsim I have made two targets:

- **riscvOVPsim_0p1**: this target basically uses the old env files and macros, which have been aliased to the new ones. There are warnings to indicate the aliasing instances when you compile. I have also ported the 64-bit tests for this one
- **riscvOVPsim** : Is my attempt to create a compliance_model.h for ovpsim using the new macros. I did it for spike, but for ovpsim the tests pass (i.e. signatures match) but the execution log keeps saying "Test FAILED". Not sure how to fix this.

Let me know if you face issues. Feel free to make a PR as well.
